src/components/auth/Login.jsx

```diff
@@ -51,6 +109,7 @@ export function Login() {
           >
             Create New Account
           </Link>
+          {error !== "" && <Error err={error} setError={setError} />}
```

Comment:
Suggestion:
Instead of checking for empty string, You can check with falsy value (!error).  Ignore if not needed.